src/bot/functions/rotate-barrel-to.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(degreeAngle, callback) {
   }
 
   // set velocity (with sign for direction)
-  this.barrel.rotationVelocity = this.maxRotationVelocity;
+  this.barrel.rotationVelocity = this.barrel.maxRotationVelocity;
```

Comment:
Previously was using the tank `maxRotationVelocity` of `.0004`

`barrel.maxRotationVelocity` is `.0009`. I really like the faster speed.